Incident_Process.md

```diff
@@ -0,0 +1,97 @@
+
+# The MacAdmins Slack Incident Process
+
+Our brief governing philosophy: 
+
+*Please strive to resolve the situation by yourself; if you think that isn’t possible or appropriate, please contact an Admin immediately.*
+
+## Something is Off
+
+A significant part of the MacAdmins Slack Admins job is the management of incidents on the MacAdmins Slack, but ideally the community (that’s you) has both the tools and the desire to resolve many situations without Admin support. 
+
+*As the MacAdmins Slack is a mature community, we find that the community naturally highlights CoC violations and they are resolved organically. If you are not comfortable with personally raising any violations of the CoC, please go ahead and jump to “It’s Admin O’Clock”.*
+
+The MacAdmins Code of Conduct (“CoC”) exists to give members a model idea of what is and isn’t acceptable in MacAdmins Slack including behaviour, marketing activity, and etiquette. If you experience strange behaviour in a channel and are wondering if it’s acceptable, please first consult the CoC.
+
+With CoC clarity in hand, please contact the member(s) and explain:
+
+1. This is the activity I saw which I have an issue with,
+2. It’s not allowed by the CoC, and,
+3. Let’s figure out how to resolve it. 
+
+Ideally, the above discussion can help resolve the situation. It often does. If does not, if you are not comfortable contacting the member(s), or if this is an obvious heinous situation, jump to “ It’s Admin O’Clock”.
+
+## It’s Admin O’Clock
+
+Administrators can be contacted directly ([Current List of Administrators](https://github.com/macadminsdotorg/slack-assets/blob/master/Admins.md)) or in the #ask-about-this-slack channel. Please choose the path most comfortable for you. 
```

Comment:
This could be a direct link to [#ask-about-this-slack](https://macadmins.slack.com/archives/C06PS0JKA)

Reply:
Good point. Added.